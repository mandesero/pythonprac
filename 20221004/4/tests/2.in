"x", "x**2+1", "x/y"
100
