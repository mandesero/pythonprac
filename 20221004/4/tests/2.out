0.009999000099990002
